app/models/listing.py

```diff
@@ -23,15 +24,19 @@ class Listing(db.Model):
     description = db.Column(db.Text)
     price = db.Column(db.Float)
     available = db.Column(db.Boolean, default=True)
+    product_id=db.Column(db.Integer, default=1)
+    updated=db.Column(db.Integer, default=0)
 
-    def __init__(self, vendor_id, name, available, category_id, price,
-                 description=""):
+    def __init__(self, product_id, vendor_id, name, available, category_id, price,
+                 description="", updated=0):
+        self.product_id = product_id
         self.vendor_id = vendor_id
         self.category_id = category_id
         self.name = name
         self.description = description
         self.price = price
         self.available = available
+        self.updated = updated
```

Comment:
set up an enum for this...or just comment in what each updated value corresponds to